cds_dojson/marc21/fields/books/values_mapping.py

```diff
@@ -40,10 +40,9 @@
 }
 
 ACQUISITION_METHOD = {
-    'user': ['H'],
+    'user': ['H', 'r'],
```

Comment:
not sure the meaning of this, but I guess it might be clear in other part of the code. If not, maybe add a comment?

Reply:
`# mapping for possible value of "created_by.type" field`